tests/integration-tests/migrations_test.go

```diff
@@ -0,0 +1,86 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright freiheit.com*/
+
+package integration_tests
+
+import (
+	"testing"
+
+	"github.com/freiheit-com/kuberpult/pkg/db"
+)
+
+func deleteSchemaMigrationsTable(cfg db.DBConfig) error {
+	db, err := db.GetDBConnection(cfg)
+	if err != nil {
+		return err
+	}
+	_, err = db.Exec("DROP TABLE schema_migrations")
+	if err != nil {
+		return err
+	}
+	return nil
+}
+func TestMigrations(t *testing.T) {
+	dbConfig := db.DBConfig{
+		DbHost:         "localhost",
+		DbPort:         "5432",
+		DbUser:         "postgres",
+		DbPassword:     "mypassword",
+		DbName:         "kuberpult",
+		DriverName:     "postgres",
+		MigrationsPath: "../../database/migrations/postgres",
+	}
+	dbHandler, err := db.Connect(dbConfig)
+	if err != nil {
+		t.Fatalf("Error establishing DB connection: %v", err)
+	}
+	pErr := dbHandler.DB.Ping()
+	if pErr != nil {
+		t.Fatalf("Error pinging database: %v", err)
+	}
+	if err := deleteSchemaMigrationsTable(dbConfig); err != nil {
+		t.Fatalf("Failed to delete schema migrations table: %v", err)
+	}
+	testCases := []struct {
+		name        string
+		postRunFunc func(db.DBConfig) error
+	}{
+		{
```

Comment:
these 2 elements don't seem to be independent test cases.
So I would suggest to have just 1 element here, and in the actual test (line ~73), to run `RunDBMigrations` twice.